packages/site-kit/components/Checkbox.svelte

```diff
@@ -0,0 +1,59 @@
+<script>
+	export let checked;
+</script>
+
+<input type="checkbox" bind:checked />
+
+<style>
+	input[type='checkbox'] {
```

Comment:
The attribute selector shouldn't be needed since it's the only input inside the component?